netsuite/10.0/modules/ROOT/pages/netsuite-studio-configure.adoc

```diff
@@ -0,0 +1,277 @@
+
+[[configure-input-source]]
+== Configure an Input Source
+
+An input source initiates a flow when a specified condition is met.
+You can configure one of these input sources to use with Netsuite:
+
+* `HTTP Listener`, which initiates a flow each time it receives a request on the configured host and port
+* `Scheduler`, which initiates a flow when a time-based condition is met
+
+For example, to configure HTTP Listener, follow these steps:
+
+. In the *Mule Palette* view, select *HTTP > Listener*.
+. Drag *Listener* to the Studio canvas.
+. On the *Listener* configuration screen, optionally change the value of the *Display Name* field.
+. Specify a value for the *Path* field.
+. Click the plus sign (+) next to the *Connector configuration* field to configure a global element that can be used by all instances of `HTTP Listener` in the app.
+. On the *General* tab, specify connection information.
+. On the *TLS* tab, optionally specify TLS information.
+. On the *Advanced* tab, optionally specify reconnection information, including a reconnection strategy.
+. Click *Test Connection* to confirm that Mule can connect with the specific server.
+. Click *OK*.
+
+[[add-connector-operation]]
+== Add a Connector Operation to the Flow
+
+When you add a connector operation to your flow, you immediately define a specific operation for that connector to perform.
+
+To add an operation for Netsuite, follow these steps:
+
+. In the *Mule Palette* view, select *Netsuite* and then select the desired operation.
+. Drag the operation onto the Studio canvas to the right of the input source.
+
+[[configure-global-element]]
+== Configure a Global Element for the Connector
+
+When you configure a connector, it’s best to configure a global element that all instances of that connector in the app can use. Configuring a global element requires you to provide the authentication credentials the connector uses to access the target Netsuite system.
+
+To configure the global element for Netsuite Connector, follow these steps:
+
+. Select the name of the connector in the Studio canvas.
+. In the configuration screen for the operation, click the plus sign (+) next to the *Connector configuration* field to access the global element configuration fields.
+. On the *General* tab, select an authentication mechanism and configure it as described:
++
+[%header,cols="30s,70a"]
+|===
+|Configuration |Description
+|Request Based Authentication |Using this strategy, you can send user credentials in the SOAP header of each request instead of authenticating to NetSuite by invoking login.
+|Token Authentication |Similar to Single Sign-On Login Authentication, this is a token based authentication system. However, instead of using a privateKey generated via openssl, this uses a consumer and token key/secret pairs which are established within the NetSuite environment.
```

Comment:
Single Sign-On Login Authentication does not exist anymore, so this reference should be removed.